src/effects/VST3/VST3Effect.h

```diff
@@ -179,4 +181,12 @@ class VST3Effect final : public StatefulPerTrackEffect
    bool FetchSettings(VST3EffectSettings& settings) const;
 
    bool StoreSettings(const VST3EffectSettings& settings) const;
+
+   static inline VST3EffectSettings& GetSettings(EffectSettings& settings)
```

Comment:
Do not make `GetSettings` a static function, until the last step to real statelessness.

Make it a non-static member function of  VST3Effect that ignores its argument and uses a VST3EffectSettings member of the VST3Effect, which is still stateful.

This would parallel what I did with AudioUnits.

This allows some changes to use `GetSettings` where that will be needed, while not yet changing real behavior, when the transformation to statelessness is still incomplete.

Reply:
done in commit `GetSettings turned to temporary version`